<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotifyConsole</class>
 <widget class="QWidget" name="NotifyConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Alerts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="EdgeFrame">
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnShowErrors">
       <property name="toolTip">
        <string>Error Alerts</string>
       </property>
       <property name="text">
        <string>Error Alerts</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnShowWarnings">
       <property name="toolTip">
        <string>Warning Alerts</string>
       </property>
       <property name="text">
        <string>Warning Alerts</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnShowInfo">
       <property name="toolTip">
        <string>Success Alerts</string>
       </property>
       <property name="text">
        <string>Success Alerts</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="BtnShowCategories">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Filter">
       <property name="placeholderText">
        <string>Filter Alerts</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnClear">
       <property name="toolTip">
        <string>Clear Resolved</string>
       </property>
       <property name="text">
        <string>Clear Resolved</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnCloseConsole">
       <property name="toolTip">
        <string>Hide Alerts and Notifications</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="WidgetsTableViewBase" name="TableIssues" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetsTableViewBase</class>
   <extends>QWidget</extends>
   <header>WidgetsModule/internal.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
